WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
Stop pretending to be dense: you know exactly what I want to change and what changes I want to make on this file. I want to fix the mistakes made in the Mercenaries file. I want to change the names to something that makes sense. The Mercenary file was hastily put together and was full of mistakes. You should be thanking me for fixing it, not blocking a very beneficial improvement.

Worse, you completely dodged the fact that you were hypocritical about the "DONAMONAPRISC_DECADENT" issue. You told me to change it because it broke your retarded Golden Rule. I offered to place a pound sign comment explaining it, but that wasn't good enough for you, so I eventually changed it.

Then, when I make the Mercenaries file actually make sense, you presented the exact same case I did, _in defiance of your Golden Rule_, and then have the audacity to say it's different.

Thus, you are a hypocrite.